exonum-java-binding-cryptocurrency-demo/pom.xml

```diff
@@ -94,6 +94,14 @@
           </execution>
         </executions>
       </plugin>
+
+      <plugin>
```

Comment:
Shan't we add a comment explaining that we skip deployment of internal modules, which is inherited from the parent?